### PR TITLE
Add HotelTable component

### LINKED ISSUE #12: Add HotelTable component (table for index/details)

### PR DESCRIPTION
**Pull request by GlassBricks**
In this PR, we add the HotelTable component for displaying hotels (closes #12).

This depends on #11; perhaps we should rebase this branch after that is merged.

### CODE REVIEW

**Comment by TheRealJBomb** [comment]
Rebase this branch with main and then LGTM to merge.

edit: not ready yet actually

**Comment by GlassBricks** [comment]
Still need to add the storybook page.

File: frontend/src/main/components/Hotels/HotelTable.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        console.log(`editCallback: ${showCell(cell)})`);
+        navigate(`/hotels/edit/${cell.row.values.id}`)
+    }
+
+    const detailsCallback = (cell) => {
+        console.log(`detailsCallback: ${showCell(cell)})`);
+        navigate(`/hotels/details/${cell.row.values.id}`)
+    }
+
+    const columns = [
```
Comment (TheRealJBomb) [inline frontend/src/main/components/Hotels/HotelTable.js:32]: I think we need a column for the _address_ field as well in the table.

**Comment by TheRealJBomb** [comment]
LGTM

**Pull request merged**